# Teensy bridge parameters
teensy_bridge:
  ros__parameters:
    serial_port: /dev/ttyACM0
    baud_rate: 115200
    wheelbase: 0.1869
    wheel_radius: 0.055
    max_steering_angle: 0.4887
    odom_frame: odom
    base_frame: base_link
    publish_tf: false
    cmd_topic: /cmd_vel
    use_stamped_vel: false
    cmd_timeout_sec: 0.5
